bin/setup

```diff
@@ -14,20 +13,22 @@ FileUtils.chdir APP_ROOT do
   # Add necessary setup steps to this file.
 
   puts "== Installing dependencies =="
-  system! "gem install bundler --conservative"
```

Comment:
This is not in the new default setup file. Was this something for the docker setup?

Reply:
That came from the old baja pet rescue app, and I cannot recall why. I probably followed a blog on setting up the rails app at that time, and that was a noted step, if it wasn't included in Rails 7 by default.

Reply:
Ok. I'll leave it out for now.